server/webhook/release.go

```diff
@@ -0,0 +1,64 @@
+package webhook
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/xanzy/go-gitlab"
+)
+
+func (w *webhook) HandleRelease(ctx context.Context, event *gitlab.ReleaseEvent) ([]*HandleWebhook, error) {
+	handlers, err := w.handleChannelRelease(ctx, event)
+	if err != nil {
+		return nil, err
+	}
+	return cleanWebhookHandlers(handlers), nil
+}
+
+func (w *webhook) handleChannelRelease(ctx context.Context, event *gitlab.ReleaseEvent) ([]*HandleWebhook, error) {
+	repo := event.Project
+	res := []*HandleWebhook{}
+	message := fmt.Sprintf("### Release: **%s**\n", event.Action)
```

Comment:
Dont you think it will be better to use a slack attachment here instead of markdown?